package.json

```diff
@@ -29,5 +29,9 @@
       "packages/**/test.js",
       "packages/**/test/*.js"
     ]
+  },
+  "scripts": {
+    "bootstrap": "lerna bootstrap",
+    "test": "ava"
```

Comment:
Okay, I suggest to use `ava && eslint .`

There are also integration tests with prosody but it may be overwhelming to install prosody for newcommers so let's leave it out from the package.json ?